src/datasetManifest.js

```diff
@@ -250,9 +250,3 @@ export function getAvailableDatasets(specificDatasets) {
   // Sort dataset names by alphabetical order.
   return availableDatasets.sort((a, b) => a.name.localeCompare(b.name));
 }
-
-export function getDefaultLabelForDataset(datasetId) {
-  return allDatasets.find(dataset => {
-    return dataset.id === datasetId;
-  });
-}
```

Comment:
Hmmm -  there are levels in the curriculum where a default label is set (preference, for example, in the food datasets. How do we do that without this code?

Reply:
Looks like we set it here: https://github.com/code-dot-org/ml-playground/blob/main/src/jsonReaderWrapper.js#L38